### PR TITLE
setup and mobile first

### PR DESCRIPTION
**Pull request by sulaimanahmadi**
1. use of Html and CSS
2. Implement the button interactions
3. The website should be responsive
4. Use of flexbox

### CODE REVIEW

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @sulaimanahmadi 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
- [ ] Please consider Updating the Microverse logo and text with yours if possible or remove it totally. The template is designed by Microverse for us all to use to professionally document our projects and has it filled with content related to each project we may be working on.
- [ ] Please consider removing non-important contents from your readme, The template is designed by Microverse for us all to use to document our projects professionally and has it filled with content related to each project we may be working on. currently, I can see you have not removed the guiding contents  which start with `>` and `directions set by Microverser`

You have submitted a project that has linter tests falling. Please make sure all linters are set up and passing before your next review request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,286 @@
...
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
```
[the remark this answers is not in the document]
Comment (sulaimanahmadi) [reply README.md:103]: Here for me it does not show me any errors regarding linters.
Screenshot_2023-03-24-19-42-42-041_com android chrome

File: style.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+.details > p {
+  max-width: 566px;
+}
+
+.social-media {
+  display: flex;
+  gap: 22.3px;
+  margin-bottom: 130px;
+}
```
Comment (Tufahel) [inline style.css:109]: - [ ] You can see, the `icon` section has more space on the left. Kindly decreased the spaces to match the given design.

- Your design:
- image

- Given design:
- image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+h2 {
+  color: #fff5e1;
+  font-family: "Roboto", sans-serif;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: left;
+}
```
Comment (Tufahel) [inline style.css:84]: - [ ] There is more gaps/padding/margin between `h1` and `h2` tags in your design, that doesn't match with the actual design. Kindly fix that issue by decreasing gaps.

- Your design:
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @sulaimanahmadi  :wave: 

Well done. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ Readme is professional.


## Required Changes ♻️

> Kindly check the inline comments for review.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — COMMENTED** [review]
Hi @sulaimanahmadi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**